src/gui/QuickSetup.hh

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#ifndef IGNITION_GAZEBO_GUI_QUICKSETUPHANDLER_HH_
+#define IGNITION_GAZEBO_GUI_QUICKSETUPHANDLER_HH_
+
+#include <QtCore>
+#include <QDesktopServices>
+#include <string>
+
+#include "ignition/gazebo/EntityComponentManager.hh"
+#include "ignition/gazebo/Export.hh"
+
+namespace ignition
+{
+namespace gazebo
+{
+// Inline bracket to help doxygen filtering.
+inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
+namespace gui
+{
+/// \brief Class for handling about dialog
+class QuickSetupHandler : public QObject
```

Comment:
By the way, more than a "quick setup" dialog, here we're going for a "quick start" dialog.

The difference being that on "setup" you choose your persistent preferences such as a dark theme or the language of the UI. While "start" is something that you may want to do multiple times, choosing something different each time, like choosing an example world to explore.

We may end up adding setup options to the dialog, but the initial focus and main goal is to offer users a "quick start". So I think we should rename all classes and files in this PR to `QuickStart`.

Reply:
make sense :+1: its not called quick start instead